app/src/main/java/org/mozilla/fenix/library/bookmarks/edit/EditBookmarkFragment.kt

```diff
@@ -185,17 +185,14 @@ class EditBookmarkFragment : Fragment(R.layout.fragment_edit_bookmark) {
                         launch(Main) {
                             Navigation.findNavController(requireActivity(), R.id.container)
                                 .popBackStack()
-                            activity.getRootView()?.let { rootView ->
-                                bookmarkNode?.let {
-                                    FenixSnackbar.make(rootView, FenixSnackbar.LENGTH_SHORT)
-                                        .setText(
-                                            getString(
-                                                R.string.bookmark_deletion_snackbar_message,
-                                                it.url?.toShortUrl(context.components.publicSuffixList) ?: it.title
-                                            )
-                                        )
-                                        .show()
-                                }
+
+                            bookmarkNode?.let {
+                                BrowserSnackbarPresenter(activity.getRootView()!!).present(
+                                    getString(
```

Comment:
Nit: there are enough layers here that I think it would help clarity if we named `it` as `node`.

Reply:
Changed 😄